### DISCUSSION TITLE
Quick question: how do you log how Rollup is resolving modules?

### DISCUSSION THREAD

**Discussion by matthew-dean** [rollup/rollup#5347] [Q&A]
I'm trying to figure out how/why Rollup is including the wrong module (not the one installed in node_modules). I tried setting `logLevel: 'debug'` in my Rollup config, but this did nothing. How can you trace module resolution?

**Comment by lukastaegert** [comment]
Write a simple plugin.You can hook into https://rollupjs.org/plugin-development/#resolveid to see what Rollup tries to resolve. Inside that hook, you can await https://rollupjs.org/plugin-development/#this-resolve to see what Rollup would have resolved to.
